courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>Maybe have only one BATCH_SIZE parameter for both train and eval (it's set at 1000 in both cases here anyway) to decrease parameter complexity. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>